package.json

```diff
@@ -229,9 +230,8 @@
 		"start": "npm run dev",
 		"prestorybook:build": "npm run build",
 		"storybook:build": "storybook build -c ./storybook -o ./storybook/build",
-		"prestorybook:dev": "npm run build",
-		"storybook:dev": "concurrently \"npm run dev\" \"storybook dev -c ./storybook -p 50240\"",
-		"storybook:e2e:dev": "concurrently \"npm run dev\" \"storybook dev -c test/storybook-playwright/storybook -p 50241\"",
+		"storybook:dev": "concurrently \"npm run dev\" \"wait-on .dev-ready && storybook dev -c ./storybook -p 50240\"",
+		"storybook:e2e:dev": "concurrently \"npm run dev\" \"wait-on .dev-ready && storybook dev -c test/storybook-playwright/storybook -p 50241\"",
```

Comment:
I don't like that typescript is forcing us to have this marker. I wonder if we can simplify and run things sequentially or something.

Reply:
Yeah, it could be possible to break down the dev/build scripts into smaller scripts. That itself may have complexity trade-offs though.